src/AppInstallerSharedLib/AppInstallerStrings.cpp

```diff
@@ -175,6 +175,32 @@ namespace AppInstaller::Utility
         return result;
     }
 
+    std::optional<std::wstring> TryConvertToUTF16(std::string_view input, UINT codePage)
+    {
+        if (input.empty())
+        {
+            return {};

```

Comment:
This returns an empty optional, which isn't what I would expect.  It should return an optional with an empty string in it.